src/components/OverviewCard.jsx

```diff
@@ -11,9 +23,131 @@ import {
   retrieveResourceFileMetadata,
 } from '../utils';
 
+// axios settings
+axios.defaults.withCredentials = true
+
 const OverviewCard = ({
   date, category, settingsToggle, itemIndex, siteName, fileName, isResource, isHomepage
 }) => {
+  const dropdownRef = useRef(null)
+  const fileMoveDropdownRef = useRef(null)
+  const categoryInputRef = useRef(null)
+  const [allCategories, setAllCategories] = useState()
+  const [newCategory, setNewCategory] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+  const [canShowDropdown, setCanShowDropdown] = useState(false)
+  const [canShowFileMoveDropdown, setCanShowFileMoveDropdown] = useState(false)
+  const [canShowDeleteWarningModal, setCanShowDeleteWarningModal] = useState(false)
+  const baseApiUrl = `${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}${category ? isResource ? `/resources/${category}` : `/collections/${category}` : ''}`
+
+  useEffect(() => {
```

Comment:
The problem with locating this in the `OverviewCard` is that when we load the Workspace or a collection, we make one API call for each `OverviewCard`, which is a waste of resources.

<img width="1415" alt="Screenshot 2020-11-16 at 5 24 55 AM" src="https://user-images.githubusercontent.com/31984694/99197239-260b0480-27cc-11eb-911c-a0d6d0b1623d.png">

If possible, it might be better to retrieve this data at the parent element and pass it as props to the `OverviewCard`. Alternatively, we can also retrieve this data whenever someone lands on the Workspace/ a Collection page / Resources / a Resource category and store it in a context. I'm keen to hear what you think - we can discuss this over a short call

Reply:
Resolved in https://github.com/isomerpages/isomercms-frontend/pull/219/commits/ee49cfca993446bad0ca259e11dfc91ea943ddd7.

As discussed, we will be retrieving the collections data in the parent for now. In future we could consider storing the state as a global level variable if concurrency issues are not significant.